### ISSUE TITLE
bug: handling of prefixed properties is inconsistent

### ISSUE THREAD

**Issue by kristerkari** [konstantin24121/stylelint-performance-animation#4]
This should be allowed, but gives a warning:

```css
.foo {
  transition-property: -webkit-transform;
}
```

This does not give a warning:

```css
.foo {
  transition: -webkit-transform 2ms linear;
}
```

This should be allowed, but gives a warning:

```css
.foo {
  transition-property: -webkit-text-stroke-color;
}
```

This does not give a warning:

```css
.foo {
   transition: -webkit-text-stroke-color 2ms linear;
}
```

/ping @konstantin24121
